StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsSessionManifest.swift

```diff
@@ -126,6 +126,84 @@ struct FinancialConnectionsSessionManifest: Decodable {
         !livemode
     }
 
+    init(
```

Comment:
I'm not _too_ sure why, but in order to initialize this in the unit tests, I had to add this explicit initializer 🤷‍♂️

Reply:
just purely for curiosity, this is very interesting haha

I wonder if its `private let _theme: Theme?` - where the `private` is somehow confusing the compiler - but its still weird because you'd think Swift currently has an `internal` initializer generated (and it works?) so why the problem? 🤔

Reply:
Actually, yeah that's it! If I get rid of the `_theme` property, the initializer is generated... Wonder why 🤔

Reply:
So it seems like I could either keep the explicit initializer or make `_theme` no longer `private`. I lean towards the former, but I'll go with whichever you prefer!

Reply:
oh no worries at all, lets go with your preference - I was just curious (it is interesting!)